docs/index.rst

```diff
@@ -37,6 +37,8 @@ Serialization API
 
 Supporting classes
     - :ref:`Message <pythonclient_message>`
+    - :ref:`NewTopic <pythonclient_newtopic>`
```

Comment:
you could perhaps put these right at the end and grouped under Admin API

```
    - Admin API
        - :ref:`NewTopic <pythonclient_newtopic>`
        - :ref:`NewPartitions <pythonclient_newpartitions>`
```

Reply:
though not sure how that looks. thinking they would be better grouped anyway.

Reply:
Thanks Matt for the reviewing, I'm working on the it:)